Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -104,6 +109,34 @@ static void webkit_user_content_manager_class_init(WebKitUserContentManagerClass
             g_cclosure_marshal_VOID__BOXED,
             G_TYPE_NONE, 1,
             WEBKIT_TYPE_JAVASCRIPT_RESULT);
+
+    /**
+     * WebKitUserContentManager::script-message-with-reply-received:
+     * @manager: the #WebKitUserContentManager
+     * @script_message_reply: the #WebKitScriptMessageReply holding the value received from the JavaScript world.
```

Comment:
This needs to be updated, the value received is @js_result and @script_message_reply is used only to send the reply